articles/2016-09-01-semantic-html-tags.md

```diff
@@ -0,0 +1,88 @@
+# :name_badge: Semantic HTML Elements
+
+For this article, I assume you have a basic working knowledge of HTML. If this is not the case, please [email me](mailto:maddie.horowitz@addepar.com) and I will bring you up to speed.
```

Comment:
Do you really want people emailing you about this? Seems like you should just link to a beginners tutorial on MDN or something.